src/dashboard/src/components/accounts/backends.py

```diff
@@ -0,0 +1,10 @@
+from shibboleth.backends import ShibbolethRemoteUserBackend
+from tastypie.models import ApiKey
+
+
+class CustomShibbolethRemoteUserBackend(ShibbolethRemoteUserBackend):
+    def configure_user(self, user):
```

Comment:
`configure_user` is a hook from the Shibboleth library which is called just after user creation.